### PR TITLE
Small osdeps fixes

### PR DESCRIPTION
**Pull request by doudou**

### CODE REVIEW

File: rock.osdeps [hunk 1]
```diff
@@ -136,3 +138,10 @@ thin: gem
 grape: gem
 sprockets: gem
 
+qtruby:
+    default:
+        gem: qtbindings
+        osdep: [qt4, qt4-opengl, qt-designer, qt4-webkit]
```
Comment (goldhoorn) [inline rock.osdeps]: Do we really need the designer for people who dosn't want to design her own guis?

Comment (doudou) [inline rock.osdeps]: No, but so far these are designed with the intent of getting a developer environment ready for people. In addition, this PR is really about doing the same-than-before. If you want to change the status quo, best do it in another PR.

Comment (D-Alex) [inline rock.osdeps]: the designer is installing some dev-libraries needed for the vizkit ui loading mechanism

**Comment by goldhoorn** [comment]
then ack for merge
